src/OmniSharp.Host/WasmCompositionHostBuilder.cs

```diff
@@ -0,0 +1,195 @@
+using System;
+using System.Collections.Generic;
+using System.Composition.Hosting;
+using System.Composition.Hosting.Core;
+using System.Linq;
+using System.Reflection;
+using Microsoft.CodeAnalysis;
+using Microsoft.Extensions.Caching.Memory;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.DependencyInjection.Extensions;
+using Microsoft.Extensions.DependencyModel;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using OmniSharp.Eventing;
+using OmniSharp.FileSystem;
+using OmniSharp.FileWatching;
+using OmniSharp.Mef;
+using OmniSharp.MSBuild.Discovery;
+using OmniSharp.Options;
+using OmniSharp.Roslyn;
+using OmniSharp.Roslyn.Utilities;
+using OmniSharp.Services;
+
+namespace OmniSharp
+{
+    public class WasmCompositionHostBuilder
```

Comment:
I mostly copy-pasted this and then started removing things we didn't need.  Probably a bit more shared code here that we can abstract out with the regular CompositionHostBuilder, but I don't want to do that until we get to a working workspace as I expect we'll find more.